analysis/scons.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import subprocess
+
+def main():
+    # Pass on command line args
+    del sys.argv[0]
+    cl_args = ' '.join(sys.argv)
+    
+    # Create call
+    call = 'python ../config/scons/scons.py %s' % cl_args
```

Comment:
Shall we make the path an argument?

Reply:
Not relevent, switching to symlink, but a good suggestion if it stayed relevant.